class.jetpack-cli.php

```diff
@@ -1015,6 +1017,11 @@ public function partner_provision( $args, $named_args ) {
 			$request_body['wpcom_user_id'] = $named_args['wpcom_user_id'];
 		}
 
+		// override email of selected user
+		if ( isset( $named_args['wpcom_user_email'] ) && ! empty( $named_args['wpcom_user_email'] ) ) {
```

Comment:
We could just check `! empty( $named_args['wpcom_user_email'] )` here. Not a blocker though.